src/LinuxTracing/ThreadStateManager.h

```diff
@@ -47,10 +47,12 @@ namespace orbit_linux_tracing {
 class ThreadStateManager {
  public:
   void OnInitialState(uint64_t timestamp_ns, pid_t tid,
-                      orbit_grpc_protos::ThreadStateSlice::ThreadState state);
-  void OnNewTask(uint64_t timestamp_ns, pid_t tid);
+                      orbit_grpc_protos::ThreadStateSlice::ThreadState state,
+                      pid_t was_blocked_by_thread = 0u, pid_t was_blocked_by_process = 0u);
```

Comment:
I don't think these two parameters can ever be not zero, as we simply read the state from the `/proc` filesystem. They should probably be removed.